src/ucp/rndv/rndv_rtr.c

```diff
@@ -433,7 +433,7 @@ ucp_proto_rndv_rtr_mtype_query(const ucp_proto_query_params_t *params,
 
     attr->is_estimation  = 1;
     attr->max_msg_length = remote_attr.max_msg_length;
-    attr->lane_map       = UCS_BIT(rpriv->lane) | remote_attr.lane_map |
+    attr->lane_map       = UCS_BIT(rpriv->lane) |
```

Comment:
`ucp_proto_rndv_mtype_query_lane(params)` is not needed. Also you always enable progress for memtype eps in this PR

Reply:
Fixed by https://github.com/openucx/ucx/pull/8986/commits/52ae79261c2c565efd0fec0701bc1012129e98f3